src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -25,8 +25,8 @@ public static class StandardTrainersCatalog
         /// <param name="featureColumnName">The features, or independent variables.</param>
         /// <param name="exampleWeightColumnName">The name of the example weight column (optional).</param>
         /// <param name="numberOfIterations">The maximum number of passes through the training dataset; set to 1 to simulate online learning.</param>
-        /// <param name="initialLearningRate">The initial <a href="tmpurl_lr">learning rate</a> used by SGD.</param>
-        /// <param name="l2Regularization">The L2 weight for <a href='tmpurl_regularization'>regularization</a>.</param>
+        /// <param name="learningRate">The learning rate. Large value may boost the training process but also leads to numerical instability.</param>
```

Comment:
>learningRate [](start = 25, length = 12)

Actually, this does mean the initial learning rate. The trainer dynamically rescales the learning rate, so it could be useful to keep calling it the initial learning rate.

Reply:
I took a short search for the uses of learning rates in other packages. It looks like no one is using initial learning rate.
TF: https://www.tensorflow.org/api_docs/python/tf/train/AdamOptimizer
Scikit-learn: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html#sklearn.linear_model.SGDClassifier